apps/main/src/dao/components/ExternalLinkIconButton.tsx

```diff
@@ -5,24 +5,25 @@ import { ExternalLink } from '@ui/Link'
 import TooltipButton from '@ui/Tooltip/TooltipButton'
 
 type ExternalLinkIconButtonProps = {
-  href: string
+  href?: string
   tooltip: string
   children?: ReactNode
 }
 
-const ExternalLinkIconButton = ({ href, tooltip, children }: ExternalLinkIconButtonProps) => (
-  <TooltipButton
-    clickable
-    noWrap
-    tooltip={tooltip}
-    customIcon={
-      <StyledExternalLink href={href}>
-        {children}
-        <Icon name="Launch" size={16} />
-      </StyledExternalLink>
-    }
-  />
-)
+const ExternalLinkIconButton = ({ href, tooltip, children }: ExternalLinkIconButtonProps) =>
+  href && (
```

Comment:
Yeah ok so not my proudest 'fix', did the same for `InfoLinkBar`. But:

1. I didn't want to modify each usage of both components with a check first
2. Some of these usages were in neat arrow body notation and I didn't want to introduce a single variable with `return` statement just for that.